mtp_api/apps/mtp_auth/views.py

```diff
@@ -59,6 +59,10 @@ def get_queryset(self):
         return queryset
 
 
+class JobInformationViewSet(viewsets.mixins.CreateModelMixin, viewsets.GenericViewSet):
+    serializer_class = JobInformationSerializer
```

Comment:
might be worth putting in an explicit `permission_classes = (IsAuthenticated,)` in here even though it's on by default cos the serialiser requires a proper user object